src/index.js

```diff
@@ -2,8 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import {BrowserRouter as Router} from 'react-router-dom';
+import ScrollToTop from './utils/ScrollToTop';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <Router>
+    <ScrollToTop>
```

Comment:
you don't need to wrap your App in ScrollToTop component. There is nothing wrong with it, just a suggestion..

Reply:
You can just return null if you decide not to wrap it

Reply:
yeah, thanks for suggestion